src/components/editors/SceneEditor.tsx

```diff
@@ -739,14 +767,14 @@ export const SceneEditor = ({ id }: SceneEditorProps) => {
               </SidebarColumn>
             )}
 
-            {colorsEnabled && (
               <SidebarColumn>
                 <FormRow>
                   <FormField
                     name="playerSpriteSheetId"
                     label={
                       <>
-                        {l10n("FIELD_SCENE_BACKGROUND_PALETTES")}
```

Comment:
Now that there's a palette to edit even in monochrome mode, I moved the `colorsEnabled` check after the label. I had a strong urge to update the indentation everywhere, but I left it alone to avoid a massive unreadable diff. There are a few bits that looked too odd without any indentation changes, but it's probably not a bad idea to fully correct the indentation levels for consistency after the bulk of the code review is done.